Alliance.Server/Extensions/AdminMenu/Handlers/AdminMenuHandler.cs

```diff
@@ -270,11 +270,11 @@ public bool SendWarningToPlayer(NetworkCommunicator peer, AdminClient admin)
 			if (playerSelected == null) return false;
 
 			GameNetwork.BeginModuleEventAsServer(playerSelected);
-			GameNetwork.WriteMessage(new SendNotification($"Vous avez reçu un avertissement d'un Admin ({peer.UserName}) !", 0));
+			GameNetwork.WriteMessage(new SendNotification($"{admin.WarningMessageToPlayer} (Admin : {peer.UserName}) !", 0));
```

Comment:
Maybe you should handle the case when there is no message inside `admin.WarningMessageToPlayer`.
In order to have a default message.

Reply:
Done : 
Added : admin.WarningMessageToPlayer = string.IsNullOrEmpty(admin.WarningMessageToPlayer) ? "Vous avez reçu un avertissement d'un Admin" : admin.WarningMessageToPlayer;